arrow/locales.py

```diff
@@ -92,7 +145,11 @@ def describe(self, timeframe, delta=0, only_distance=False):
 
         return humanized
 
-    def describe_multi(self, timeframes, only_distance=False):
+    def describe_multi(
+        self,
+        timeframes: Collection[Tuple[TimeFrames, Union[int, float]]],
```

Comment:
Are we using a collection here to encompass both the possibility of both `tuple` and `list`? Do you think it's problematic since collections span beyond these two types?

Reply:
Fixed it to `Sequence` because after merging master, there are usage of random access on `timeframes`.